apps/desktop/src/renderer/components/app/AppShell.tsx

```
@@ -201,14 +203,17 @@ export function AppShell({ children }: { children: React.ReactNode }) {
         setProject(null);
         setProjectMissing(false);
         setShowWelcome(true);
+      } finally {
+        if (cancelled) return;
```

Biome (2.4.9)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)